app/views/help/cookie_settings.html.haml

```diff
@@ -0,0 +1,59 @@
+- if params[:"analytical-cookies"] == "off"
+  .lcdui-notice-summary
+    .govuk-error-summary
+      .govuk-error-summary__title
+        Analytical cookies turned off
+- if params[:"analytical-cookies"] == "on"
+  .lcdui-notice-summary
+    .govuk-error-summary
+      .govuk-error-summary__title
+        Analytical cookies enabled
+
+= govuk_page_title(t('cookie_settings.page_title', service_name: service_name))
+
+%p.govuk-body= t('cookie_settings.section_intro.paragraph_one')
+
+%p.govuk-body= t('cookie_settings.section_intro.paragraph_two')
+
+%h2.govuk-heading-l= t('cookie_settings.section_necessary.heading')
+
+%p.govuk-body= t('cookie_settings.section_necessary.paragraph_one')
+
+%ul.govuk-list.govuk-list--bullet
+  %li= t('cookie_settings.section_necessary.list_item_one')
+  %li= t('cookie_settings.section_necessary.list_item_two')
+
+%p.govuk-body= t('cookie_settings.section_necessary.paragraph_two')
+
+%p.govuk-body
+  %a.govuk-link{ :href => "cookies" }= t('cookie_settings.section_necessary.link_to_cookie_page')
+
+.vcd-no-js-only
+  %h2.govuk-heading-l= t('cookie_settings.section_settings.heading_no_js')
+
+  %p.govuk-body= t('cookie_settings.section_settings.paragraph_one_no_js')
+
+  %p.govuk-body= t('cookie_settings.section_settings.paragraph_two_no_js')
+
+%form#cookie-form.vcd-js-only{:method => "get", :data => {:module => "cookie-settings"}}
```

Comment:
Standard: use the new json style hash syntax instead of the rocket hash syntax

Reply:
Done 👍